src/App.tsx

```diff
@@ -1,7 +1,13 @@
+import { useState } from "react";
 import "./App.css";
+import initialData from "./assets/tasks-example.json";
 import TableHeader from "./components/TableHeader";
 
 function App() {
+  const [tasks, setTasks] = useState(initialData);
```

Comment:
setTasks is never used.

Reply:
Yes, but we'll need it when we want to edit the tasks.